manifest.json

```diff
@@ -1,39 +1,38 @@
 {
   "manifest_version": 3,
   "name": "Better Youtube Search",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Improve Youtube search results.",
-   "permissions": ["scripting"],
+  "permissions": ["scripting", "tabs"],
```

Comment:
When I was using the chrome.tabs api, I didn't need the `tabs` permission to check the URL. However, when I initially used the browser.tabs API, I did need the tabs permission. I switched back to chrome.tabs because Google Chrome didn't recognize the `browser` namespace, although Firefox recognizes the `chrome` namespace. This directly contradicts the [Mozilla docs on different API implementations](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Differences_between_API_implementations) for the browser extension API.